### PR TITLE
Content elements

### PR DESCRIPTION
**Pull request by frett**
moved paragraph, form, and tabs to content schema

### CODE REVIEW

**Comment by frett** [comment]
this is the schema update, I'm working on updating the pages right now

**Comment by frett** [comment]
all the existing content has been updated for this PR

File: public/xmlns/content.xsd [hunk 1]
```diff
@@ -118,6 +118,29 @@
...
+    <xs:complexType name="paragraph">
+        <xs:choice minOccurs="1" maxOccurs="unbounded">
+            <xs:group ref="content:elements" />
+        </xs:choice>
+    </xs:complexType>
+
+    <!-- Tabbed content -->
+    <xs:complexType name="tabs">
+        <xs:sequence>
+            <xs:element name="tab" minOccurs="1" maxOccurs="unbounded">
```
Comment (ryancarlson) [inline public/xmlns/content.xsd]: Didn't we mean to put a max on this? I know we said 2 as a "soft" max, but we probably should enforce a hard max at 3 or no more than 4.

Comment (frett) [inline public/xmlns/content.xsd]: yeah, we had talked about it, I'll go toss one on. We can always adjust it later :)

File: public/xmlns/content.xsd [hunk 2]
```diff
@@ -128,7 +128,7 @@
     <!-- Tabbed content -->
     <xs:complexType name="tabs">
         <xs:sequence>
-            <xs:element name="tab" minOccurs="1" maxOccurs="unbounded">
+            <xs:element name="tab" minOccurs="1" maxOccurs="3">
```
Comment (ryancarlson) [inline public/xmlns/content.xsd]: 👍

**Pull request merged**